Streetcode/Streetcode.WebApi/Extensions/ConfigureHostBuilderExtensions.cs

```diff
@@ -30,4 +32,19 @@ public static void ConfigureInstagram(this IServiceCollection services, WebAppli
     {
         services.Configure<InstagramEnvirovmentVariables>(builder.Configuration.GetSection("Instagram"));
     }
+
+    public static void ConfigureSerilog(this IServiceCollection services, WebApplicationBuilder builder)
+    {
+        var filterExpression = builder.Configuration["Serilog:Filter:ByExcluding"];
```

Comment:
Does this work or do we use the custom implementation?

Reply:
it work, but only for pathes